storage/api/build.gradle

```diff
@@ -4,6 +4,11 @@ dependencies {
     implementation project(':ethereum:pow:api')
     implementation project(':ethereum:spec')
 
+    implementation 'org.rocksdb:rocksdbjni'
+    implementation 'org.fusesource.leveldbjni:leveldbjni-win64'
+    implementation 'org.fusesource.leveldbjni:leveldbjni-win32'
+    implementation 'tech.pegasys:leveldb-native'
```

Comment:
hmm, doesn't seem right that the API module depends on these implementation details.

Reply:
As discussed, will attempt to move logic out of DatabaseVersion to elimnate this problem